docs/distributed/README.md

```diff
@@ -43,6 +43,8 @@ To start a distributed MinIO instance, you just need to pass drive locations as
 - Servers running distributed MinIO instances should be less than 15 minutes apart. You can enable [NTP](http://www.ntp.org/) service as a best practice to ensure same times across servers.
 - Running Distributed MinIO on Windows operating system is experimental. Please proceed with caution.
 - `MINIO_DOMAIN` environment variable should be defined and exported if domain is needed to be set.
+- The number of drives needs to be a multiple of 4, 6, 8, 10, 12, 14 or 16.
```

Comment:
```suggestion
- The number of drives needs to be divisible by either 4, 6, 8, 10, 12, 14 or 16.
```

Reply:
@harshavardhana 'multiple' is easier for me to understand, but it still has that weirdness where any multiple of 8 is always a multiple of 4. 

I assume you want to say that the drives are split into erasure sizes of "4, 6, 8, 10, 12, 14 or 16".

Maybe a section like:

> Erasure sets of 4, 6, 8, 10, 12, 14 or 16 are created, depending on the number of drives added. By default the biggest possible erasure set is created.

I don't know enough about the specifics if that is correct, but if that is what you want to tell make it something like that.

Reply:
Yes that makes sense..